pilz_psenscan_test_area/setup_workspace.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
+
+create_catkin_ws()
+{
+  mkdir -p "$SRC_DIR" 2>&1
+}
+
+clone_repo()
+{
+  ls "$REPO_DIR" > /dev/null 2>&1 && rm -rf "$REPO_DIR" 2>&1
```

Comment:
In case the repo exists, I as user still expect to get _no_ error message because it is not a real error.  Therefore, I suggest to write `ls "$REPO_DIR" > /dev/null 2>/dev/null && rm -rf "$REPO_DIR" 2>&1`

Reply:
No error is shown in the case you mentioned